app/client/components/transferAccount/transferAccountManager.jsx

```diff
@@ -326,6 +341,13 @@ class TransferAccountManager extends React.Component {
               </Option>
             </Select>
           </Descriptions.Item>
+          {this.state.last_known_card_balance ? (
+            <Descriptions.Item label="Last Known Card Balance">
```

Comment:
This means a zero balance on the card won't display.

Reply:
Fixed!